### ISSUE TITLE
Make it possible for render_bundle to generate script tags with the async attribute

### ISSUE THREAD

**Issue by jjinux** [owais/django-webpack-loader#63]
In our own code, we created a helper called render_async_bundle that does the same thing as render_bundle, but adds an async attribute to the script tag. This is quite helpful in certain situations.

**Comment by owais** [comment]
Just released `0.3.2`. You can now specify a string to append to the tags. Example,
```html+django
{% render_bundle 'main' 'js' attrs='async charset="UTF-8"'%}
```
will add `async` and `charset="UTF-8"` attrs to the tag

**Comment by jjinux** [comment]
Awesome! Thanks!
